src/me/StevenLawson/TotalFreedomMod/Commands/Command_opall.java

```diff
@@ -32,16 +30,23 @@ else if (args[0].equals("-s"))
         }
 
         for (Player player : server.getOnlinePlayers())
-        {
-            player.setOp(true);
-            player.sendMessage(TFM_Command.YOU_ARE_OP);
+        {	
+        	if (!player.isOp())
+        	{
+        		TFM_Util.adminAction(sender.getName(), "Opping all players on the server", false);
```

Comment:
You didn't test this at all, did you?
Please **compile and test** you changes **before** making a PR.